imports/plugins/included/connectors-shopify/server/methods/import/customers.js

```diff
@@ -166,9 +166,11 @@ export const methods = {
 
             // Insert customer, save id
             const reactionCustomerId = Accounts.insert(reactionCustomer, { publish: true });
+            Hooks.Events.run("afterAccountsInsert", reactionCustomerId, reactionCustomerId);
```

Comment:
The first arg should be `Meteor.userId()` instead of `reactionCustomerId`. It is supposed to be the user who is triggering the operation, i.e. user who called the server method.